src/pages/Integrations/Create/CardSelect.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { FormGroup, FormHelperText, Grid, GridItem, HelperText, HelperTextItem, Tile } from '@patternfly/react-core';
+
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+
+/**Temporarily copied from sources-ui
+ * This component will soon be imported to component-groups
+*/
+
+const handleKeyPress = (event, value, onClick) => {
+  const spaceBar = 32;
+  if (event.charCode === spaceBar) {
+    event.preventDefault();
+    onClick(value);
+  }
+};
+
+const CardSelect = (originalProps) => {
+  const { isRequired, label, helperText, hideLabel, meta, input, options, mutator, DefaultIcon, iconMapper, ...props } =
+    useFieldApi(originalProps);
+  const formOptions = useFormApi();
+  const [icons] = useState(() => {
+    const components = {};
+
+    options.forEach(({ value }) => (components[value] = iconMapper(value, DefaultIcon)));
+
+    return components;
+  });
+
+  const isMulti = props.isMulti || props.multi;
+  const isDisabled = props.isDisabled || props.isReadOnly;
+  const inputValue = input.value || [];
+
+  const handleMulti = (value) =>
+    input.onChange(
+      inputValue.includes(value) ? inputValue.filter((valueSelect) => valueSelect !== value) : [...inputValue, value]
+    );
+
+  const handleSingle = (value) => input.onChange(inputValue === value ? undefined : value);
+
+  const handleClick = (value) => (isMulti ? handleMulti(value) : handleSingle(value));
+
+  const onClick = (value) => {
+    if (isDisabled) {
+      return undefined;
+    }
+
+    handleClick(value);
+    input.onBlur();
+  };
+
+  const prepareCards = () =>
+    options
+      .map((option) => mutator(option, formOptions))
+      .map(({ value, label, isDisabled: itemIsDisabled }) => {
+        const disabled = itemIsDisabled || isDisabled;
+
+        if (!value) {
+          return undefined;
+        }
+
+        const Component = icons[value];
+
+        return (
+          <GridItem sm={6} md={4} key={value}>
+            <Tile
+              className={`src-c-wizard__tile${disabled ? ' disabled' : ''}`}
```

Comment:
also `src-c-wizard__tile` does not exist - you can use it from sources - cardSelect.scss